pkg/apis/scheduling/types.go

```diff
@@ -59,6 +59,9 @@ const (
 type PodGroupConditionType string
 
 const (
+	// PodGroupUnInqueueable is Uninqueueable event type
+	PodGroupUnInqueueable PodGroupConditionType = "Uninqueueable"
```

Comment:
suggest change `uninqueueable` to `unenqueueable` and `PodGroupUnInqueueable` to `PodGroupUnenqueueable`.  `inqueue` is a noun which stands for a state.

Reply:
Has been done.